hydroflow/src/compiled/pull/half_join_state/multiset.rs

```diff
@@ -5,10 +5,18 @@ use std::collections::{hash_map::Entry, VecDeque};
 
 type HashMap<K, V> = rustc_hash::FxHashMap<K, V>;
 
+use smallvec::smallvec;
+use smallvec::SmallVec;
```

Comment:
Nit: Combine the two uses

Reply:
I wonder why cargo fmt didn't combine them or complain about it?